### PR TITLE
Added loadgeneratorFloodHomepage flag description to docs

### PR DESCRIPTION
**Pull request by bornav**
Adding documentation https://github.com/open-telemetry/opentelemetry-demo/pull/1486

### CODE REVIEW

**Comment by cartermp** [comment]
@bornav do you still intend for this to be a draft?

**Review by julianocosta89 — APPROVED** [review]

**Comment by julianocosta89** [comment]
This one should be good to go! 🥳

File: content/en/docs/demo/feature-flags.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ values are stored in the `demo.flagd.json` file. To enable a flag, change the
 | `recommendationServiceCacheFailure` | Recommendation   | Create a memory leak due to an exponentially growing cache. 1.4x growth, 50% of requests trigger growth.  |
 | `paymentServiceFailure`             | Payment Service  | Generate an error when calling the `charge` method                                                        |
 | `paymentServiceUnreachable`         | Checkout Service | Use a bad address when calling the PaymentService to make it seem like the PaymentService is unavailable. |
+| `loadgeneratorFloodHomepage`        | Loadgenerator    | Start flooding the homepage with a huge amount of requests, configurable by changing flagd JSON on state  |
```
Comment (theletterf) [inline content/en/docs/demo/feature-flags.md:25]:
```suggestion
| `loadgeneratorFloodHomepage`        | Loadgenerator    | Start flooding the homepage with a huge amount of requests, configurable by changing flagd JSON on state. |
```

Missing period

File: content/en/docs/demo/feature-flags.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ values are stored in the `demo.flagd.json` file. To enable a flag, change the
 | `recommendationServiceCacheFailure` | Recommendation   | Create a memory leak due to an exponentially growing cache. 1.4x growth, 50% of requests trigger growth.  |
 | `paymentServiceFailure`             | Payment Service  | Generate an error when calling the `charge` method                                                        |
```
Comment (theletterf) [inline content/en/docs/demo/feature-flags.md:23]:
```suggestion
| `paymentServiceFailure`             | Payment Service  | Generate an error when calling the `charge` method.                                                       |
```

**Review by theletterf — APPROVED** [review]

**Pull request merged**